src/Chapter1.hs

```diff
@@ -508,8 +511,8 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y = if (abs x) < (abs y) then x else y
+  --min (abs x) (abs y)
```

Comment:
That is a nice guess! 👍🏼 
However, this solution doesn't preserve sign, so for negative numbers, it still will return positive.